odometry_loop_rate: 50.0 # [Hz]
home_position_publish_rate: 1.0 # [Hz]

# px4 # #{
px4:
  param_set_timeout: 3 # [s] Minimal value is 1. If zero, the parameter will never be set
  EKF2_AID_MASK: 1 # default mask - use only gps

  # Default source of height estimation  - will be changed according to sensor reliability:
  # 0 - only barometer mode,
  # 1 - only gps,
  # 2 - rangefinder height mode
  EKF2_HGT_MODE: 2
  EKF2_RNG_AID: 1 # Using the range sensor to estimate height if specific conditions are met
  EKF2_RNG_A_HMAX: 5.0 # Max height to use rangefinder for fusion in aid mode

# # #}
